decoderx/http.go

```diff
@@ -518,7 +509,7 @@ func (t *HTTP) decodeURLValues(values url.Values, paths []jsonschemax.Path, o *h
 						continue
 					}
 
-					raw, err = sjson.SetBytes(raw, path.Name, v)
+					raw, err = sjson.SetRawBytes(raw, path.Name, []byte(v))
```

Comment:
As we expect an object in this case, we can assume the value is premarshalled. The validation happening later will catch violations of this assumption.

Reply:
That makes sense to me